read_repair_test.py

```diff
@@ -794,6 +791,61 @@ def test_atomic_writes(self, repair_type, expect_atomic):
             else:
                 assert listify(results) == [kcvv(1, 0, 2, 1)]
 
+class TestSpeculativeReadRepairLongerTimeout(Tester):
+
+    @pytest.fixture(scope='function', autouse=True)
+    def fixture_set_cluster_settings(self, fixture_dtest_setup):
+        cluster = fixture_dtest_setup.cluster
+        cluster.set_configuration_options(values={'hinted_handoff_enabled': False,
+                                                  'dynamic_snitch': False,
+                                                  # longer timeout to reduce the likelihood of a speculative write
+                                                  'write_request_timeout_in_ms': 4000,
+                                                  'read_request_timeout_in_ms': 1000})
+        cluster.populate(3, install_byteman=True, debug=True)
+        byteman_validate(cluster.nodelist()[0], './byteman/read_repair/sorted_live_endpoints.btm', verbose=True)
+        cluster.start(wait_for_binary_proto=True, jvm_args=['-XX:-PerfDisableSharedMem'])
+        session = fixture_dtest_setup.patient_exclusive_cql_connection(cluster.nodelist()[0], timeout=2)
+
+        session.execute("CREATE KEYSPACE ks WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3}")
+        session.execute("CREATE TABLE ks.tbl (k int, c int, v int, primary key (k, c)) WITH speculative_retry = '400ms';")
+
+    @since('4.0')
+    def test_speculative_data_request_without_speculative_write(self):
```

Comment:
replicated the above test here, but in this class the cluster is set up with a longer write_request_timeout_in_ms, because we want to assert that speculated_rr_write is equal to 0